frontend/src/utils/api/reviews.js

```diff
@@ -28,6 +28,18 @@ export function reviewMapper({ id, code, likes, userID, title, body, timestamp }
   }
 }
 
+export function newReviewMapper({ title, body, recommend, enjoy, difficulty, teaching, workload }) {
+  return {
+    title,
+    body,
+    recommend: {'Yes': 1, 'No': 0}[recommend],
```

Comment:
Confused about the {}[] syntax, is this matching the variable in [] to a key in the {} to set the value?